packages/multicast_dns/lib/src/resource_record.dart

```diff
@@ -0,0 +1,182 @@
+// Copyright 2018 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:meta/meta.dart';
+
+/// Enumeration of support resource record types.
+class RRType {
```

Comment:
nit: not sure it applies in this case, but, avoid abbreviations unless they are more recognisable than the full name